roles/appsuite/templates/opt/open-xchange/etc/findbasic.properties.j2

```diff
@@ -0,0 +1,18 @@
+{{ ansible_managed | comment }}
+
+# Some mail backends provide a virtual folder that contains all messages of
+# a user to enable cross-folder mail search. Open-Xchange can make use of
+# this feature to improve the search experience.
+#
+# Set the value to the name of the virtual mail folder containing all messages.
+# Leave blank if no such folder exists.
+com.openexchange.find.basic.mail.allMessagesFolder = Virtual/All
+
+# Denotes if mail search queries should be matched against mail bodies.
+# This improves the search experience within the mail module, if your mail
+# backend supports fast full text search. Otherwise it can slow down the
+# search requests significantly.
+#
+# Change the value to 'true', if fast full text search is supported. Default
+# is 'false'.
+com.openexchange.find.basic.mail.searchmailbody = true
```

Comment:
Please make this one a template (using `| ternary ('true', 'false')`) and set a default of `false` in the role defaults.